src/code-generation/CodeGeneratorBase.ts

```diff
@@ -98,14 +98,27 @@ abstract class CodeGeneratorBase<TSettings extends CodeGeneratorBaseSettings> im
     --this._branches.length
     return this
   }
+  wackIndent(oneLine: boolean): this {
```

Comment:
This method seems to be only relevant for your specific use case. Move it to your class.